sdks/python/apache_beam/yaml/tests/map.yaml

```diff
@@ -31,8 +31,8 @@ pipelines:
             append: true
             fields:
               # TODO(https://github.com/apache/beam/issues/32832): Figure out why Java sometimes re-orders these fields.
-              literal_int: 10
               named_field: element
+              literal_int: 10
```

Comment:
We should try to get to the bottom of this... File a bug?

Reply:
Agreed, this isn't super sustainable. I think we can just use https://github.com/apache/beam/issues/32832